blocks/subscriptions-block/components/OfferToProductList/index.jsx

```diff
@@ -32,21 +35,26 @@ const OfferToProductList = ({ offer, isLoggedIn, checkoutURL, loginURL, classNam
 						actionText: strategies[strategiesIdx].summary,
 						actionEvent: () => {
 							Sales.clearCart()
-								.then(() =>
+								.then(() => {
 									Sales.addItemToCart([
 										{
 											sku,
 											priceCode,
 											quantity: 1,
 										},
-									])
-								)
+									]);
+									const allValidUntil = offer?.campaigns?.map(c => c.validUntil !== undefined && !Number.isNaN(c.validUntil));
+									const maxEndDate = allValidUntil.length ? Math.max(allValidUntil) : null;
+									const liveCampaing = offer?.campaigns?.find(c => c.validUntil === null || c.validUntil === maxEndDate);
+									localStorage.setItem(ARCXP_CAMPAIGN, liveCampaing?.name);
```

Comment:
The ticket says `save the offer campaign,  in session storage ArcXP_campaignName.` Do we want to store in localStorage?

Reply:
@edwardcho1231 yes, by default we are using rememberMe true for login or signUp, for this reason we need to save this info on localStorage, This if the user opens a new page, the info is availabke